### ISSUE TITLE
Create preprocessing data

### ISSUE THREAD

**Issue by okids** [okids/projekEPL#3]
-The data must be scaled in case of using gradient descent algorithm or artificial neural network. 
-The data must follow normal distribution. If it isn't following normal distribution we must doing any mathematical operation.

**Comment by okids** [comment]
-We also must filter what features that affect our data target. And what if we combine the features using mathematical operation, it is have better correlation with data target?
